docs/license-manager-setup.md

```diff
@@ -0,0 +1,60 @@
+# Set up License Manager for dedicated hosts
```

Comment:
Do we need to include information in the main setup guide to point to this? Otherwise people aren't going to know that this is necessary to configure Mac tests